locks.go

```diff
@@ -0,0 +1,24 @@
+package appconfig
+
+import "fmt"
+
+type Lock struct{}
+
+// LockKV locks the key/value pair at key, optionally with a
+// label. If you don't want to pass a label, pass an empty
+// string.
+//
+// Returns an error if the lock couldn't be acquired,
+// and returns the lock itself if the lock could be acquired
+func (a *Client) LockKV(key, label string) (*Lock, error) {
```

Comment:
Is `KV` the same thing as "Key" when we do `GetAllKeys`? The Keys that we see/create in the Azure Portal? If so, my vote would be to call this `LockKey` instead.

If not, I need to do more reading. Sorry for the dumb questions, still learning the API! 😀